### PR TITLE
fix(react-components): adopt Cogs chips and fix 360 image details panel appearance

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change

Chore 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:

~~Migrate components according to Cogs.js adoption plan.~~

This grew a bit out of it's original plans, but the change seems sort-of necessary: The Image360 panel would not always show up when entering 360 panels because of some React re-rendering shenanigans - the 360 image container is temporarily unmounted after getting added, and we did not properly react to when it was added anew. This PR fixes the second problem - the 360 is still mounted twice, but we now correctly react to it (by using the `use360Collections` hook, and making sure the resource count includes 360 images). 

Aside from that, I've simplified two occurrences of cogs.js chips as part of the cogs-adaption.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [ ] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/CadModelContainer/CadModelContainer.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export function CadModelContainer({
     addModel(addModelOptions, transform)
       .then((model) => {
         onLoad?.(model);
-        setRevealResourcesCount(viewer.models.length);
+        setRevealResourcesCount(viewer.models.length + viewer.get360ImageCollections().length);
```
Comment (danpriori) [inline react-components/src/components/CadModelContainer/CadModelContainer.tsx:47]: I would prefer to use a const variable to set this summed counter, but it is more a personal taste, TBH. We can keep this as it is, if you prefer :)

**Review by danpriori — APPROVED** [review]
one comment but LGTM

File: react-components/src/components/CadModelContainer/CadModelContainer.tsx (hunk 1 above)
Comment (haakonflatval-cognite) [reply react-components/src/components/CadModelContainer/CadModelContainer.tsx:47]: I agree that this should be more standardized across different model types... I added a `getViewerResourceCount` function that is used on all locations

File: react-components/src/components/CadModelContainer/CadModelContainer.tsx [hunk 2]
```diff
@@ -44,7 +45,7 @@ export function CadModelContainer({
     addModel(addModelOptions, transform)
       .then((model) => {
         onLoad?.(model);
-        setRevealResourcesCount(viewer.models.length);
+        setRevealResourcesCount(getViewerResourceCount(viewer));
```
Comment (danpriori) [inline react-components/src/components/CadModelContainer/CadModelContainer.tsx:48]: Super! Way better IMO :)

**Pull request merged**